src/api/roles/shared/chat.py

```
@@ -0,0 +1,83 @@
+    new_message = ChatMessage(chat_id=chat_id, from_account_id=acc.id, message_text=message_text, is_read=False)
+    db.add(new_message)
+    db.flush()
+    db.commit()
+
+    if new_message.id is None:
+        raise HTTPException(500, detail="Message creation failed")
+
+    return SendMessageResponse(message_id=new_message.id, message_text=new_message.message_text, from_account_id=new_message.from_account_id)
+
+@router.get("/get_messages/{chat_id}", response_model=GetMessagesResponse)
+def get_messages(chat_id: int, pagination: PaginationParams = Depends(PaginationParams), db = Depends(get_session), acc: Account = Depends(get_client_account)):
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)